EXOSIMS/PlanetPhysicalModel/uniform_albedo.py

```diff
@@ -0,0 +1,49 @@
+from EXOSIMS.util.vprint import vprint

```

Comment:
This class doesn't need to exist.  Let's port the functionality to the prototype's calc_albedo_from_sma method, and just ensure that the prototype's prange is [0.367, 0.367] (or whatever the exact number in the current method is) to protect backwards compatibility.